### DISCUSSION TITLE
completed lesson 1

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#487] [Show and tell]
GM!

I just finished lesson 1 and I'm feeling pretty good about it! It was a lot of information to take in, but the material was presented in a clear and concise manner. I'm excited to learn more and see where this knowledge can take me.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/75005151/63f59022-73d5-413d-98b9-ed04152c8990)

**Comment by 0xusmanf** [comment]
Good luck and keep it up.

**Comment by dodeyeikoi** [comment]
hey guys I just finished lesson !, I know I've only just began haha

**Reply by 0xusmanf** [reply]
Great, keep up!

**Comment by SquilliamX** [comment]
Great job! welcome to smart contract development! let's change the world together! :)
